handbook/engineering/core-application/index.md

```diff
@@ -4,49 +4,43 @@
 
-Our current work-streams are:
+## Members 
 
-1. **Core application**: Building, securing and scaling our multi-tenant hosted version of Sourcegraph for customers that do not want to deploy Sourcegraph on-premise. Also meeting the needs of larger and larger enterprises to support those deals (e.g. Perforce support)
-2. **Backend platform**: Make it easy for teammates of different experience levels to onboard and contribute to backend code. Hide and remove non essential complexity from engineers working on new product features and use cases. Promote and champion consistency and simplicity across all backend code by building shared tools, libraries and infrastructure for common use cases. Scale and maintain said infrastructure. Create leverage for and accelerate other teams.
+<!-- Due to the markdown renderer that we use, the indentation here is sensitive. If you want to change the indentation, check that it renders correctly locally with `make serve` -->
 
-## Areas of ownership
+- [Ryan Phillips](../../company/team/index.md#ryan-phillips-he-him) ([Product Manager](../../product/roles/index.md#product-manager))
+- [Quinn Keast](../../company/team/index.md#quinn-keast-he-him) ([Product Designer](../../product/roles/index.md#product-designer))
+- [Rafal Leszczynski](../../company/team/index.md#rafal-leszczynski-he-him) ([Engineering Manager](../roles.md#engineering-manager)) {#core-application-eng}
+  - [Joe Chen](../../company/team/index.md#joe-chen)
+  - [Ryan Slade](../../company/team/index.md#ryan-slade-he-him)
+  - [Alan Harris](../../company/team/index.md#alan-harris-he-him)
+  - [Artem Ruts](../../company/team/index.md#artem-ruts-he-him)
+  - [Asdine El Hrychy](../../company/team/index.md#asdine-el-hrychy)
+  - [Indradhanush Gupta](../../company/team/index.md#indradhanush-gupta-he-him)
+  - Milan Freml (Aug 9)
+  - Rafal (Oct 11)
 
-- Authorization and authentication
-- Repository management (gitserver, repo-updater, src-expose)
-- Data storage and access libraries
-- GraphQL API
-- License generation and enforcement
-- Admin and user settings
-- Analytics
+## How to contact the team and ask for help
+- Please ask questions and submit requests for assistance via [#core-application](https://sourcegraph.slack.com/archives/CHPC7UX16) Slack channel 
+- Use `@core-app` to mention the team in Slack
+- Use [team/core-application](https://github.com/sourcegraph/sourcegraph/issues/new?labels=team/core-application) label to tag the team in GitHub.
```

Comment:
Do we want to mention anything about jira?

Reply:
we use Jira internally as it's not public for now. It would be hard to ask people to use it if they don't have access.